src/SiteConfig/ConfigBuilder.php

```diff
@@ -340,7 +346,6 @@ public function mergeConfig(SiteConfig $currentConfig, SiteConfig $newConfig)
             $currentConfig->replace_string[] = $replaceString;
         }
 
-        // merge http_header array from currentConfig into newConfig
         // because final values override former values in case of named keys
         $currentConfig->http_header = array_merge($newConfig->http_header, $currentConfig->http_header);
```

Comment:
I mean, you should move the whole 3 lines

Reply:
Done